### DISCUSSION TITLE
More features from Tweaks

### DISCUSSION THREAD

**Discussion by marcustyphoon** [AprilSylph/XKit-Rewritten#280] [Suggestion Box]
Tweaks for the ~~tweaks god~~ uh, project board.

I just skimmed the old list of XK7 Tweaks options, discarded any that are no longer relevant or that I don't remember or never had any idea the function of (so some may be missing), and kept the rest!

This isn't a feature suggestion so much as a "feature or list-as-unimplemented-feature" suggestion.

I would like the notification badge style one though. And I know many people really, really wanted to hide their follower counts (sounds like a job for requestAnimationFrame :D) The rest seem fairly meh.

**already done**
Make URLs grey again (actually the opposite now)
Don't show my own posts on my dashboard
Hide the post highlight that appears when using jk to scroll
Make the Follow button more subtle inside posts

**already on the consider/todo list**
Hide the mini-follow buttons on posts and notifications
Stop avatars from scrolling along with the post

**could do, at least in theory**
Show one line of tags on posts by default (really not worth it imo)
Hide the share button on posts
Use a real red on ~~notifications and~~ post like buttons (???)
Don't show liked posts on my dashboard (does anyone actually want this? I don't)
Hide the explore link at the top of the page and in the sidebar
Hide the notes on posts
Hide the tags on posts
Hide your follower count in the account menu and on the sidebar
Notification badge style (per badge?) - hidden, different colors
Turn the New Post button gray (I guess)

**Comment by arcanetrivia** [comment]
The shade of red of the "like" heart (and, I guess, the little indicator that appears on notifications on the activity page, this thing?
![image](https://user-images.githubusercontent.com/68124115/134106108-429438fa-d052-482d-8101-6b934facbf19.png)
? not honestly sure what's being referred to there) is not a "true red", it's kind of vaguely pinkish/orangish I think... but I would guess the only reason there was a tweak for this in the first place is that it used to be a different shade of red, and when Tumblr changed it people wanted to put it back. The shade of red they use kinda annoys me but it might not be worth putting a lot of development time into.

**Reply by AprilSylph** [reply]
Yeah, that Tweak was from back when Tumblr's red was more of a dark orange. I think the current red is red enough.

**Reply by arcanetrivia** [reply]
Huh funny, I remember it being a cooler/bluer red in the past and _that_ is what that tweak set it back to? Pretty specific thing though, and not really the same impact on everyday use as when they decide to change the shade of "Tumblr blue" and everyone goes apeshit :) (I use Palettes too btw.)

**Reply by AprilSylph** [reply]
No, the tweak is from when the like button was orange. The tweak made it more red, hence the wording of "real red".

Pre-2016 Tumblr Liked Like Button (Recreation) | Tweaks "Real Red" Liked Like Button
----- | -----
![Screenshot 2021-09-21 at 18-46-30 Likes Tumblr](https://user-images.githubusercontent.com/28949509/134221491-3d3da86a-5b86-4530-8756-28cb46cf2292.png) | ![index](https://user-images.githubusercontent.com/28949509/134219962-26ff65fe-b0de-4f41-a98b-c427d97e83bc.png)

Fun fact: the code for this tweak in XKit 7 has remained completely untouched since 2014.

**Comment by AprilSylph** [comment]
### Will not implement
- **Show one line of tags on posts by default**: I assume this Tweak is leftover from before Tumblr would limit the height of tags itself
- **Hide the share button on posts**: this Tweak is from before messaging existed
- **Use a real red on notifications and post like buttons**: this Tweak increased the saturation of the red on notification bubbles and like buttons, which we don't need now that Tumblr has introduced its own more saturated red
- **Hide the notes on posts**: stupid
- **Hide the tags on posts**: stupid
- **Turn the New Post button gray**: why? P.S. you can do this already by using the Nuclear White palette

### No plans to implement
- **Don't show liked posts on my dashboard**: this has been requested in one review on AMO but nobody else has mentioned it and it does seem to be a niche feature, so to reduce bloat I'm considering this one's usecase fulfilled by Painter until demand for it goes up
- **Hide the explore link at the top of the page and in the sidebar**: nobody has requested this thus far; I actually used this tweak but only to keep the number of header buttons the same after adding the XKit 7 button, which is now not there for me :)
- **Notification badge style**: would require changing how XKit Rewritten's Tweaks works, which I'm not sure is worth it for such a minor option

### To do
- **Hide the mini-follow buttons on posts and notifications**: will replace (and inherit from) the subtleness tweak
- **Stop avatars from scrolling along with the post**: will probably get done this week
- **Hide your follower count in the account menu and on the sidebar**: a CSS approach would probably work

**Reply by marcustyphoon** [reply]
> **Notification badge style:** would require changing how XKit Rewritten's Tweaks works

Is this just for the ability to do a dropdown with multiple options? All I really want is "hide the [home, I guess messages, activity] notification badge" checkboxes - color is w/e.
